R/qc_ids.R

```diff
@@ -35,14 +35,14 @@
 #' )
 #' }
 qc_ids <-
-  function(data, sample, grouping, intensity, remove_na_intensities = TRUE, condition = NULL, title = "ID count per sample", plot = TRUE, interactive = TRUE) {
+  function(data, sample, grouping, intensity = NULL, remove_na_intensities = TRUE, condition = NULL, title = "ID count per sample", plot = TRUE, interactive = TRUE) {
```

Comment:
```suggestion
  function(data, sample, grouping, intensity = NULL, remove_na_intensities = TRUE, condition = NULL, title = "ID count per sample", plot = TRUE, interactive = TRUE) {
```
why did you set `intensity = NULL`? The argument is required.